tests/api.c

```diff
@@ -65944,7 +65944,7 @@ static int test_dtls_msg_get_connected_port(int fd, word16 *port)
     XMEMSET((byte*)&peer, 0, sizeof(peer));
     len = sizeof(peer);
     ret = getpeername(fd,  (SOCKADDR*)&peer, &len);
-    if (ret != 0 || len > sizeof(peer))
+    if (ret != 0 || len > (int)sizeof(peer))
```

Comment:
Same as above, since `len` is a `XSOCKLENT`, should this cast be to a `(XSOCKLENT)`?

Reply:
Yes, that is true. Fixed.